### DISCUSSION TITLE
Open SCAD for noobs.

### DISCUSSION THREAD

**Discussion by ciege92** [kennetek/gridfinity-rebuilt-openscad#54] [Q&A]
Can someone point me to a good video on using open scad? I have never used it and opening it very much confused me. The amounts of open scad files I have seen in the 3D printing has also made me want to use it. Thank y'all.

**Comment by Ruudjhuu** [comment]
What is wrong with the first videos that pop up when you enter `OpenScad` in the search bar on youtube?
